src/panels/OriginPanel/RemainingFlightTime.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect } from 'react';
+import { Box, Group, Stack, Text } from '@mantine/core';
+
+import { useGetPropertyOwner } from '@/api/hooks';
+import { AirplaneIcon } from '@/icons/icons';
+import {
+  subscribeToCameraPath,
+  unsubscribeToCameraPath
+} from '@/redux/camerapath/cameraPathMiddleware';
+import { useAppDispatch, useAppSelector } from '@/redux/hooks';
+import { IconSize } from '@/types/enums';
+import { ScenePrefixKey } from '@/util/keys';
+
+export function RemainingFlightTime() {
```

Comment:
This name seems a bit vague and makes me wonder what type of component it is.  Maybe add the word "Indicator", or similar..?